cmd/migrate/migrate.go

```diff
@@ -48,6 +48,14 @@ func migrateDashboardWidgets(tx *gorm.DB) *gorm.DB {
 		}
 	}
 
+	templates = []models.DashboardTemplate{}
+
+	// remove all non default templates
+	res = tx.Unscoped().Where(`"default" is false`).Delete(&[]models.DashboardTemplate{})
+	if res.Error != nil {
+		return res
+	}
+
```

Comment:
We have hundreds of them because every time reset is called a new layout is created. Until we will allow multiple templates per dashboard (not happening today), we can safely remove everyhting.